apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -2882,7 +2882,6 @@ public static String retrieveFDADrugCodeDisplay(String claimDrugCode) {
             String.format("%s-%s", nationalDrugCodeManufacturer, nationalDrugCodeIngredient),
             ndcProductColumns[3] + " - " + ndcProductColumns[13]);
       }
-      ndcProductsIn.close();
```

Comment:
Was this change to `TransformerUtils` intended?

Reply:
Yes. I'm cleaning up an unnecessary close. The try-resource block does the close.